libqtile/config.py

```diff
@@ -610,3 +640,75 @@ def __repr__(self):
         actions = utils.describe_attributes(self, ['group', 'float',
             'intrusive', 'break_on_match'])
         return '<Rule match=%r actions=(%s)>' % (self.match, actions)
+
+
+class DropDown(configurable.Configurable):
+    """
+    Configure a specified command and its associated window for the SratchPad.
+    That window can be shown in current group and hidden again at keystroke.
```

Comment:
`can be shown and hidden using a configurable keystroke or any other scripted trigger.`